PSI_Project/Models/Comment.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace PSI_Project.Models;
+
+public class Comment : BaseEntity 
+{
+    public string TopicId { get; init; }
+    public string CommentText { get; init; } 
+
+    public Comment(string topicId, string commentText) : base()
```

Comment:
Don't need to call empty base(); constructor and in other classes, because modified baseEntity does not require the name field anymore, so constructor does not need the name, and the base class is being called implicitly anyway, so this code "base();" is no longer needed anywhere.